packages/requisitions/src/RequestRequisition/DetailView/RequestLineEdit/RequestLineEditForm.tsx

```diff
@@ -1 +1,30 @@
-export const ResponseLineEditForm = () => {};
+import React from 'react';
+import { ItemRowFragment, ItemSearchInput } from '@openmsupply-client/system';
+import { useRequestRequisitionLines } from '../../api';
+
+interface RequestLineEditFormProps {
+  item: ItemRowFragment | null;
+  disabled: boolean;
+  onChangeItem: (item: ItemRowFragment) => void;
+}
+
+export const RequestLineEditForm = ({
+  item,
+  disabled,
+  onChangeItem,
+}: RequestLineEditFormProps) => {
+  const { lines } = useRequestRequisitionLines();
+  return (
+    <ItemSearchInput
```

Comment:
oh. unknown type `ConnectorError` etc.
I think merge this PR and then do the API changes